platform/platform-app/src/main/resources/features.xml

```diff
@@ -764,6 +752,7 @@
              description="Schedules tasks">
         <feature prerequisite="true">commons</feature>
         <feature prerequisite="true">security-core-api</feature>
+        <feature prerequisite="true">security-core</feature>
```

Comment:
I thought we were moving away from the prerequisite=true in our feature files. Is that coming later or did we find we still needed them?